src/components/localization/localized-link.js

```diff
@@ -188,7 +183,7 @@ const ExternalLink = ({
     const { is_eu_country } = useContext(DerivStore)
     const { setModalPayload, toggleModal } = useContext(LocationContext)
     const { affiliate_lang } = language_config[locale]
-    const url = getURLFormat(type, replaceLocale(locale), to, affiliate_lang)
```

Comment:
What is the reason that we no longer need this?

Reply:
it was replacing the Chinese urls with a hyphen rather than an underscore, giving 404 errors when we select some trading platforms in these languages.

Reply:
Could you check with @nikolaiberestevich-deriv on this? I remembered he did this to fix an issue with language codes coming from backend.

Reply:
sure, I will

Reply:
yes , I had an issue  (see the parent task of https://redmine.deriv.cloud/issues/46240#Broken_links_for_%22zh_tw%22_or_%22zh_cn%22_languages) maybe the problem is that some of apps( trading platforms) use "_" and our deriv.com uses "-".